native/cocos/core/scene-graph/Scene.h

```diff
@@ -52,6 +52,8 @@ class Scene final : public Node {
     void onBatchCreated(bool dontSyncChildPrefab) override;
     bool destroy() override;
 
+    int32_t getNewReflectionProbeId() { return _reflectionProbeId++; }
```

Comment:
Please add const.

Reply:
_reflectionProbeId need to be ++

Reply:
Why do it like this? It is wired.

Reply:
And please add inline for functions implemented in header file.

Reply:
when add a reflection probe to a scene, new id needs to be generated, these codes sync ts engine